.circleci/docker/Dockerfile

```diff
@@ -30,13 +30,13 @@ ENV TORCH_CUDA_ARCH_LIST Maxwell
 ENV TORCH_NVCC_FLAGS "-Xfatbin -compress-all"
 ENV CUDA_PATH /usr/local/cuda
 
-# Whether to build torch and torch_xla libraries with CXX ABI
+# Set C/C++ compilers
 ENV CC "${cc}"
 ENV CXX "${cxx}"
-ENV CXX_ABI "${cxx_abi}"
+
+# Whether to build torch and torch_xla libraries with CXX ABI
 ENV _GLIBCXX_USE_CXX11_ABI "${cxx_abi}"
-ENV CFLAGS "${CFLAGS} -D_GLIBCXX_USE_CXX11_ABI=${CXX_ABI}"
-ENV CXXFLAGS="${CXXFLAGS} -D_GLIBCXX_USE_CXX11_ABI=${CXX_ABI}"
+ENV CXXFLAGS "-D_GLIBCXX_USE_CXX11_ABI=${cxx_abi}"
```

Comment:
hmm, do we still need to manually set `"-D_GLIBCXX_USE_CXX11_ABI` here? I was hoping by setting `_GLIBCXX_USE_CXX11_ABI` env var above will build pytorch with the correct CXXABI. and then pytorch/xla will just read pytorch setting in our `setup.py`

Reply:
Unfortunately, we do -- if we want to use `torch._C._GLIBCXX_USE_CXX11_ABI`, then we should set `CXXFLAGS`, too -- otherwise, it always returns 1 (True) alwyas -- [whelp :(](https://discuss.pytorch.org/t/setting-glibcxx-use-cxx11-abi-to-0/98054)

Reply:
any reason we overwrite the `CXXFLAGS` instead of pending to it? I guess it is base env of the docker so there shouldn't be any `CXXFLAGS` before this?

Reply:
Good catch, it worked -- but to be future-proof, let me append :) thanks!